packages/@okta/vuepress-site/docs/guides/sign-into-web-app/main/index.md

```diff
@@ -24,90 +24,48 @@ This guide explains how to use Okta as the user store for your server-side web a
 * Knowledge of building server-side Web applications
 * A project or application to add authentication to.
 
-**Sample code**
-
-<StackSelector snippet="samplecode" noSelector/>
-
 ---
 
-## Refresh tokens and web apps
```

Comment:
I removed this because it's not really important for Spring Boot apps since the refresh token will be stored in the session and the session will likely timeout before the refresh token is used.